src/analysis/simulation.jl

```diff
@@ -181,6 +183,11 @@ function simulate(wm_data::Dict{String,Any}, wm_solution::Dict{String,Any},
 
     # WNTR simulation 
     wns = wntr.sim.EpanetSimulator(wn)
-    wnres = wns.run_sim()
+    path_to_tmp_folder = string(@__DIR__)*"/"*Random.randstring(5)
+    tmp_file = Random.randstring(5)
```

Comment:
@enzoliuyang94 suggested that random file naming isn't necessary with the latest WNTR/EPANET?

Reply:
@jjstickel I think the reason Byron still suggest using random name is for thread safety (in case he wants to do something in parallel in the future)? Such temporary files and folder are deleted after simulation anyway, so giving them random names shouldn't do any bad.

Reply:
I think this is a recent change from Yang's branch that got merged into mine before I submitted this pull request. Yang's right, I suggested this to ensure that everything will be thread-safe algorithmically.

Reply:
Is it necessary to have both a random folder name and a random filename?

In any case, I recommend:
```
path_to_tmp_folder = mktempdir()
tmp_file = "epanetfile" # OR `Random.randstring(5)`
wnres = wns.run_sim(joinpath(path_to_tmp_folder, tmp_file))
```

Reply:
No, I can't think of a reason why that would be necessary. Totally agree with your proposal.